### ISSUE TITLE
Regex parsing errors?

### ISSUE THREAD

**Issue by ramkumarkb** [monitoror/monitoror#249]
The sample regex expression was  and it did not seem to work: 
`"regex": "Alive Workers:\s(.*)"`
`"regex": "Alive Workers:\s\d"`

I also verified with:
https://regex-golang.appspot.com/assets/html/index.html

**Comment by jsdidierlaurent** [comment]
Hello !

Did you have an error in UI using this regex ?
Can i see your tile config ?

Because in your example, you need to escape `\` like this:
`"regex": "Alive Workers:\\s(.*)"`
`"regex": "Alive Workers:\\s\\d"`

**Comment by ramkumarkb** [comment]
Hello,

I tried your suggestion with `\` but still no luck. In the UI, I see the error related to parsing config JSON.

It works ok, if I give RegEx expression, like - `"regex": "Alive Workers:"` but does not work if I give `"regex": "Alive Workers:\\s(.*)"`

```
{
  "type": "GROUP",
  "label": "Apache Spark",
  "tiles": [
    {
      "type": "HTTP-RAW",
      "label": "Spark-Master",
      "params": {
                "url": "http://spark-master:8080/",
                "regex": "Alive Workers:\\s(.*)"
                }
        },
    {
      "type": "HTTP-STATUS",
          "label": "Spark-Worker-1",
      "params": {
                "url": "http://spark-worker-1:8081/",
                "statusCodeMin": 200, 
                "statusCodeMax": 200
      }
    },
    {
      "type": "HTTP-STATUS",
          "label": "Spark-Worker-2",
      "params": {
                "url": "http://spark-worker-2:8081/",
                "statusCodeMin": 200, 
                "statusCodeMax": 200
      }
    }
  ]
},
```

**Comment by jsdidierlaurent** [comment]
Hello !

I try to reproduce your issue with:

#### config.json
``` json
{
	"version": "1.0",
	"columns": 1,
	"tiles": [
		{"type": "HTTP-RAW", "params": {"url": "http://localhost:7777/test.txt", "regex": "Alive Workers:\\s(.*)"}}
	]
}
```

And i expose this one with simple http server

#### test.txt
```
# Test file
Alive Workers: 2

# :D
```

With this, monitoror show me:
![image](https://user-images.githubusercontent.com/11354381/76292452-8fd32600-62af-11ea-8699-b6b1bfb8ef85.png)

Can i see a sample of your http://spark-master:8080/ result ?

**Comment by oviliz** [comment]
Apologise I'm hijacking this, is there a way I can capture a change to "Everything is looking good" at https://status.dev.azure.com? I know that when there's a problem that text would change to something else.

**Comment by Alex-D** [comment]
@oviliz Please create a dedicated issue for each question/issue. It becomes hard to follow up if everyone adds his question ^^

Moreover, if anyone else searches for the same issue as you, it will be easier to find in a dedicated issue.

Thank you :)

**Comment by oviliz** [comment]
@jsdidierlaurent, your test works for me in monitoror but is not producing any results in that Regex Tester I was also referring to:
![image](https://user-images.githubusercontent.com/73616/76688556-b5e92500-6625-11ea-9676-fdf1076d6532.png)
Can you please suggest any other tool to test our regex?

**Comment by oviliz** [comment]
@Alex-D / @jsdidierlaurent , should this be a case of updating the example in the documentation with the escapes?
https://monitoror.com/documentation/#tile-http-raw

**Comment by Alex-D** [comment]
We are looking to remove the necessity to escape, which seems better to us.

If we cannot do that, yes, we will update the doc and add an explanation :)

**Comment by Alex-D** [comment]
OK, So the issue is: JSON tries to escape the s in `\s`, so we need to escape the backslash.

No way to avoid that, I will update the documentation ASAP :)

**Comment by Alex-D** [comment]
I reopen
I've added a more clear error, which is cool, but I need to update doc :)

**Comment by ramkumarkb** [comment]
> Hello !
> 
> I try to reproduce your issue with:
> 
> #### config.json
> ```json
> {
> 	"version": "1.0",
> 	"columns": 1,
> 	"tiles": [
> 		{"type": "HTTP-RAW", "params": {"url": "http://localhost:7777/test.txt", "regex": "Alive Workers:\\s(.*)"}}
> 	]
> }
> ```
> 
> And i expose this one with simple http server
> 
> #### test.txt
> ```
> # Test file
> Alive Workers: 2
> 
> # :D
> ```
> 
> With this, monitoror show me:
> ![image](https://user-images.githubusercontent.com/11354381/76292452-8fd32600-62af-11ea-8699-b6b1bfb8ef85.png)
> 
> Can i see a sample of your http://spark-master:8080/ result ?

@jsdidierlaurent It turned out to be from the HTML page. My apologies for the confusion earlier 🙏 
`<li><strong>Alive Workers:</strong> 2</li>`

So no wonder a simpler regex gave parsing errors. For it work, it had to be -
`"regex": "Alive Workers:<\/strong>\\s(.)"`

I think the new feature of highlighting the Regex Error would be really helpful.
